### ISSUE TITLE
REVERT CrewLink - Because what you've created is game breaking, and has killed the Proximity Chat community.

### ISSUE THREAD

**Issue by Forbycakes** [ottomated/CrewLink#602]
Please, for the love of god - I've reported the most recent bugs. Myself and friends have shared screenshots, and videos, and step-by-step ways to repeat the bugs we are getting.
**It's been 3 weeks and not a single useful response from the CrewLink team.**
The reason I'm frustrated is because many people have been asking you to REVERT to the previous version until this one isn't so bugged. 
You have successfully killed at least half of the Proximity Chat community across a plethora of servers, and those that haven't left just DEAL with the fact that your application is game breaking and glitchy as all heck.

TL;DR
Please, **just revert to the older (not as pretty) CrewLink.**

xo

**Comment by Mincedpickles** [comment]
Agreed!

**Comment by crsct** [comment]
I'm left speechless at this rude and incomprehensible request. You guys know this is a github repository right? He orderdly tagged his releases, dont like a version? Go back to an old one, I am sure there will be a few that are still working, and if they are not.. fork the repository and help yourself. It's licensed under the GPL, no Developer should need to listen to rude "issues" about missusing their product.

**Comment by duracell** [comment]
@crsct The point is, that the developer always say (and often also not friendly), a high cpu usage and broken sound are no bugs and that the auto-update always updates to the broken version. That people are sad about this reaction is understandable (for me). 
@Forbycakes I disabled the auto-update and put a version of this on github, maybe this helps you until the original or another developer fixed crewlink: https://github.com/duracell/CrewLink-NoAutoUpdate

**Comment by wheatpete** [comment]
I've had issues ever since we updated to 2.0.1. Meanwhile the previous version was just fine.

**Comment by wheatpete** [comment]
> I'm left speechless at this rude and incomprehensible request. You guys know this is a github repository right? He orderdly tagged his releases, dont like a version? Go back to an old one, I am sure there will be a few that are still working, and if they are not.. fork the repository and help yourself. It's licensed under the GPL, no Developer should need to listen to rude "issues" about missusing their product.

This is honestly more rude than what OP posted

**Comment by DiamondMiner88** [comment]
> This is honestly more rude than what OP posted

He's right though. Op's not request, but DEMAND is extremely rude. If he doesn't like the glitches he can use an older version or fork the repo and fix them himself.

**Comment by frymezim** [comment]
> > I'm left speechless at this rude and incomprehensible request. You guys know this is a github repository right? He orderdly tagged his releases, dont like a version? Go back to an old one, I am sure there will be a few that are still working, and if they are not.. fork the repository and help yourself. It's licensed under the GPL, no Developer should need to listen to rude "issues" about missusing their product.
> 
> This is honestly more rude than what OP posted

The developer has no fucking obligation to any of you. So ordering around like fix this and that, is extremely rude to developer. First of all he's never been paid to do this and the creation of the this program was for purely for pursuit of knowledge. If you are reading the changelog the only reason why is this happening is because the server that hosting the crewlink is probably using the old version of the server.

**Comment by DennisSmolek** [comment]
The developer may have no obligation, but his community is fracturing and he doesn’t seem to care.

there is a general consensus to switch to the “betterCrewLink” version because of the inactivity.

there are major breaking issues in the new release and the software has an auto updater. Yet once the release went out they haven’t rolled back nor even acknowledged the issue.

There are COUNTLESS garbage issues so it’s clear there is no one handling moderation..

If this was an active community I could see the comment being rude but at this point I would wonder if the project had been abandoned...

**Comment by Flova** [comment]
We just switched to [betterCrewLink](https://github.com/OhMyGuus/BetterCrewLink).

While I understand the point, that this is a hobby project and people invest lots of time into it, I also understand that the proxy chat community suffered quit a lot damage and is just restarting using betterCrewLink. And its not like anyone is requesting big features or something similar. But I call a project unmaintained if no one reverts a broken version for a few weeks.
I maintain some larger project my self in my free time and I also hate rude feature requests and missing contributions. But on the other hand we as the maintainers take some responsibility that the project is working as intended and not suddenly broken. Other people that are not as familiar use our software and sometime (more or less)  rely on it.

Sadly many people (often without the technical expertise) do not switch to forks as betterCrewLink and abandon the proxy chat a a whole.
